Dockerfile

```diff
@@ -1,22 +1,23 @@
-ARG TAG="2.6.1"
-ARG UBI_IMAGE=registry.access.redhat.com/ubi7/ubi-minimal:latest
+ARG TAG="v2.6.1"
+ARG BCI_IMAGE=registry.suse.com/bci/bci-base:latest
 ARG GO_IMAGE=rancher/hardened-build-base:v1.16.10b7
 
 # Build the project
 FROM ${GO_IMAGE} as builder
-RUN set -x \
- && apk --no-cache add \
+RUN set -x && \
+    apk --no-cache add \
     git \
     make
 ARG TAG
-RUN git clone --depth=1 https://github.com/k8snetworkplumbingwg/sriov-cni
-WORKDIR sriov-cni
-RUN git fetch --all --tags --prune
-RUN git checkout tags/${TAG} -b ${TAG} 
-RUN make clean && make build 
+RUN git clone --depth=1 https://github.com/k8snetworkplumbingwg/sriov-cni && \
+    cd sriov-cni && \
+    git fetch --all --tags --prune && \
+    git checkout ${TAG} && \
```

Comment:
@manuelbuil there are no `tag/<tag>` branches anymore so I used a git tag instead. should that be of any concern?